drivers/mil_blueview_driver/include/blueview_wrapper.hpp

```diff
@@ -35,9 +35,11 @@ class BlueViewSonar
 
   // Return a reference to the head for configuration like setting range
   BVTSDK::Head& getHead();
+  void updateHead(); // Must call after changes to head from getHead()
```

Comment:
You should add doc strings for these public methods, in blueview_ros_driver.hpp as well.